src/server/server.ts

```diff
@@ -226,23 +229,23 @@ export class GraphQLTransportWSServer {
               // result is a GraphQL ExecutionResult, which has an optional errors property
               params.callback = (error: any, result: any) => {
```

Comment:
this should be much simple..
if data ? send data with result as payload
if error ? send error with error as payload